pkg/blueprint/blueprints/redis-blueprint.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: cr.kanister.io/v1alpha1
```

Comment:
@shahpratikr this should be symlink to `examples/redis/redis-blueprint.yaml` instead of copying whole file

Reply:
fixed